TextMiningFinalProject.py

```diff
@@ -0,0 +1,199 @@
+import requests
+import pickle
+import string
+from bs4 import BeautifulSoup
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+import matplotlib.pyplot as plt; plt.rcdefaults()
+import numpy as np
+import matplotlib.pyplot as plt
+
+#importing all the books:
+communist_manifesto = requests.get('http://www.gutenberg.org/cache/epub/61/pg61.txt').text
+f = open('communist_manifesto.pickle','wb')
+pickle.dump(communist_manifesto,f)
+f.close()
+input_file = open('communist_manifesto.pickle','rb')
+reloaded_copy_of_texts1 = pickle.load(input_file)
+
+constitution = requests.get('http://www.gutenberg.org/cache/epub/5/pg5.txt').text
+f = open('constitution.pickle','wb')
+pickle.dump(constitution,f)
+f.close()
+input_file = open('constitution.pickle','rb')
+reloaded_copy_of_texts2 = pickle.load(input_file)
+
+trotsky = requests.get('http://www.gutenberg.org/cache/epub/38982/pg38982.txt').text
+f = open('trotsky.pickle','wb')
+pickle.dump(trotsky,f)
+f.close()
+input_file = open('trotsky.pickle','rb')
+reloaded_copy_of_texts3 = pickle.load(input_file)
+
+utilitarianism = requests.get('http://www.gutenberg.org/cache/epub/11224/pg11224.txt').text
+f = open('utilitarianism.pickle','wb')
+pickle.dump(utilitarianism,f)
+f.close()
+input_file = open('utilitarianism.pickle','rb')
+reloaded_copy_of_texts5 = pickle.load(input_file)
+
```

Comment:
Right now, it seems like text-fetching using `requests` and `pickle` and text analysis are done in same file. This defeats the purpose of pickling. You will be making a request to Project Gutenberg every time you run a file, which is unnecessary. What you can do is creating a separate file where you do all text fetching and storing. This way, you can just run this file once to create a local pickled copy of the text and you don't have to get a text from Project Gutenberg website every time you run an analysis.